js/validation.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * input이 공백인지 검사
+ * @param {*} target 이벤트 발생한 타겟 요소
+ * @param {*} errorMessage 에러를 표시할 <span> 태그
+ * @returns input의 입력이 공백이면 true, 공백이 아니면 false 반환
+ */
+function handleEmptyInput(target, errorMessage) {
```

Comment:
1. trim 메소드에는 인자가 필요 없습니다.` trim()` 으로만 작성해주셔도 됩니다.
2. target.dataset.valid는 항상 문자열로 저장되기 때문에 string 값("true" or "false")을 할당해주시는게 혼동스럽지 않을 것 같습니다.
3. 어차피 인풋의 empty 여부로 조건을 나누고, 해당값을 반환할거라면 isEmpty 라는 변수를 함수 첫줄에 설정하고 해당 값을 리턴하도록 수정하면 좋을 것 같습니다.
```suggestion
function handleEmptyInput(target, errorMessage) {
const isEmpty = inputField.value.trim() === "";
if(isEmpty){
...
}
else {
...
}

return isEmpty
```